pkgs/dart_services/lib/src/analysis.dart

```diff
@@ -327,7 +325,16 @@ class AnalysisServerWrapper {
         }
       } else if (import.packageImport) {
         final packageName = import.packageName;
-        if (!isSupportedPackage(packageName)) {
+
+        if (isFirebasePackage(packageName)) {
+          // TODO: Update the 'more info' url.
```

Comment:
@johnpryan - do you have a link we should use here? Either a github issue or a wiki page, likely w/ context about the features changes for the dartpad refresh.

Reply:
It looks like @parlough added details here: https://github.com/dart-lang/dart-pad/wiki/Package-and-plugin-support#deprecated-firebase-packages

Reply:
Thanks! Feel free to adjust or add to the messaging there.